.github/workflows/release.yml

```diff
@@ -35,9 +35,12 @@ jobs:
         run: npx nx build @ethnolib/language-chooser-react-mui
 
       # find-language is a dependency of langauge-chooser-react-mui so it will have been built in the build step
-      - name: Run tests
+      - name: Run unit tests
         run: npx nx run-many --all --target=test
 
+      - name: Run e2e tests
+        run: npx nx run-many --all --target=e2e
```

Comment:
Just added this, an attempt to have GHA run the e2e tests. It's not yet tested and I just added a fix in a very recent push. Is this what we meant? Am I missing something?

Reply:
Sorry, yes that's it. Not sure why I didn't see the commit before.